velox/common/memory/MemoryUsageTracker.cpp

```diff
@@ -53,7 +53,7 @@ void MemoryUsageTracker::update(int64_t size) {
       reservationBytes_ = newQuantized;
     }
   }
-  if (decrement > 0) {
+  if (decrement != 0) {
```

Comment:
What happens if decrement is less than zero?

Reply:
Yes, I have the same question...

Reply:
I had a unit test and it can happen so I just make this check to be consistent with previous code. We can revisit this after making this thread safe.